manual/Website3/package.json

```diff
@@ -47,12 +51,12 @@
     "mini-css-extract-plugin": "^0.6.0",
     "moment": "^2.24.0",
     "optimize-css-assets-webpack-plugin": "^5.0.1",
-    "react": "file:../../node_modules/react",
-    "react-dom": "file:../../node_modules/react-dom",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
```

Comment:
because of yarn workspaces, we can't guarantee the location - may be hoisted or not